hoomd/md/nlist.py

```diff
@@ -62,7 +62,7 @@ class NList(_HOOMDBaseObject):
 
     .. rubric:: Diameter shifting
 
-    Set `diameter_shift` to `True` when using `hoomd.md.pair.SLJ` or
+    Set `diameter_shift` to `True` when using
```

Comment:
Re-organize the lines in this paragraph so the end of each line is close to the 80 character limit.

Reply:
Okay! I thought one of the automatic hooks would do that

Reply:
flake8 will tell you when you go over the limit. It does not report lines that are under the limit or reflow paragraphs.